planetaryimage/pds3image.py

```diff
@@ -138,8 +153,9 @@ def _save(self, file_to_write, overwrite):
 
         if self._sample_bytes != self.data.itemsize:
             self.label['IMAGE']['SAMPLE_BITS'] = self.data.itemsize * 8
-            sample_type_to_save = self.DTYPES[self._sample_type[0] + self.dtype.kind]
-            self.label['IMAGE']['SAMPLE_TYPE'] = sample_type_to_save
+
+        sample_type_to_save = self.DTYPES[self._sample_type[0] + self.dtype.kind]
```

Comment:
While writing a test, I realized that these 2 changed lines does not belong to the if loop. (When you create a PDS3Image object from bumpy array with `sample_type` other than `MSB_INTEGER`, you'll get an error if these lines are inside the if loop.)